channels/bo.m3u

```diff
@@ -1,7 +1,15 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",ATB (HD)
+http://186.121.206.197/live/daniel/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",ATB La Paz (Backup) [-2 minutes]
```

Comment:
Well, we do not know the location of this ATB. (At the moment I was watching, they were all 3 (La Paz, Cochabamba and Santa Cruz) streaming a Disney program for kids. Since I do not know the exact location of this, I preferred adding a backup.